src/Tooltip/Tooltip.js

```diff
@@ -196,15 +227,14 @@ class Tooltip extends React.Component<AllProps, State> {
           onFocus: this.handleRequestOpen,
           onMouseLeave: this.handleRequestClose,
           onBlur: this.handleRequestClose,
-          onKeyboardFocus: open ? this.handleRequestClose : this.handleRequestOpen,
         });
       } else {
         children = childrenProp;
       }
     }
 
     return (
-      <Manager className={classNames(classes.root, classNameProp)}>
+      <Manager className={classNames(classes.root, className)} {...other}>
```

Comment:
The reason I removed `{...other`} was because `Manager` is a `div` and `React` didn't allow using unknown props with `div`. Ex: `onRequestClose`, `onRequestOpen`.